src/app/app-routing.module.ts

```diff
@@ -71,6 +73,10 @@ export const routes: Routes = [
     path: 'news/:id',
     component: EcoNewsDetailComponent
   },
+  {
```

Comment:
please return back create-news route, 'cause after resolving conflicts it was overwritten

Reply:
fixed